src/main/create-operator-json/core/operatorFactory.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+export default class Operator{
+
+    // must pass r6operator as first argumet
```

Comment:
Add a comment explaining what each argument is.

Reply:
added comments